### PR TITLE
Add and remove

### PR DESCRIPTION
**Pull request by dvcorreia1984**
Added the following in this branch: 

1. HTML
2. Add and remove functions in Javascript

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  const author = document.getElementById('Author').value;
+  if (title && author) {
+    const bookId = books.length + 1;
+    books.push({ Id: bookId, Title: title, Author: author });
+    displayBooks();
+    document.querySelector('form').reset();
+    localStorage.setItem('myBooklist', JSON.stringify(books));
+  }
+}
+addBook();
```
Comment (thecodechaser) [inline script.js:46]: - [ ] The local storage functionality is not working, On refresh I can't see my added books on the display. Please make sure to fix this issue.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @dvcorreia1984, @redtah, and @HossainAraf👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+function removeBook(removeId) {
+  const filter = books.filter((remBook, i) => {
+    if (removeId === remBook.Id) {
+      books.splice(i, 1);
+      displayBooks();
+    }
+    return true;
+  });
+  return filter;
+}
```
Comment (binhussen) [inline script.js:58]: - [Optional] Great Job for updating to add data to local storage, I think you forget to delete also from local storage when data is removed which leads getting deleted data on refresh, so update the local storage on each remove operation to fix this issue 👍

**Review by binhussen — APPROVED** [review]
Hi @dvcorreia1984 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Created Add & Remove Functionality ✔️
- Well Structured Code ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**